src/ui/Abbreviation.js

```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 /**
- * Used to wrap a label in abbreviation tags if a valid tooltip is provided. We differentiate the abbr style
- * depending on the type passed in so that the dotted line is shown properly on the metric labels.
+ * Used to wrap a label in abbreviation tags if a valid tooltip is provided.
  */
 const Abbreviation = ({
```

Comment:
Is there much value in offering this as a component in bandit-pouch? Seems to just wrap an html element and not add much more functionality. <abbr> is styled automatically by bootstrap.

Reply:
@twheys I don't have a strong case for that one. It's used in a couple of places in Bandit, but I guess we can leave that out. I'll remove it.